.github/workflows/test.yml

```diff
@@ -46,7 +46,11 @@ jobs:
           cache: yarn
 
       - run: yarn install --immutable --inline-builds
-      - run: yarn lint
+      - run: |
+          yarn lint:eslint
+          yarn lint:prettier
+          # v4.0.0 になったら有効化する
+          # yarn lint:styleilnt
```

Comment:
After making changes to yarn.lock, a stylelint error occurred despite not modifying any .tsx files. This issue was not expected to occur since, in version 4.0.0, the files were replaced with .css and development was conducted with stylelint checks without any problems.